### PR TITLE
Fixing readme for the Universal NRP adapter

### PR DESCRIPTION
**Pull request by kkanas**
## Typo fixes and update missing file

Documentation fixes, no code was changed and no code behaviour was altered

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [x] All unit tests are passing.
- [x] I have merged the latest `main` branch prior to this PR submission.
- [x] I submitted this PR against the `main` branch.

### CODE REVIEW

**Comment by kkanas** [comment]
@microsoft-github-policy-service agree

File: src/adapters/mc/README.md [hunk 1]
```diff
@@ -15,16 +15,16 @@ To regenerate code, see [codegen.cmd](codegen.cmd).
...
 The OSConfig Universal NRP binary is built on Ubuntu 14 with gcc 4.8 in order for this same binary to run as-is on all newer Linux distros.
 
 Install Ubuntu 14.04 LTS from one of the archived locations such as at [Ubuntu](https://www.releases.ubuntu.com/14.04/), or [Ubuntu MATE](https://releases.ubuntu-mate.org/archived/14.04/amd64/)). 
 
-Install the miminal set of dependencies necessary:
+Install the minimal set of dependencies necessary:
 
 ```bash
-sudo apt-get install -y gcc git cmake build-essential
+sudo apt-get install -y gcc git cmake3 build-essential
```
Comment (MariusNi) [inline src/adapters/mc/README.md:27]: >cmake3  [](http://example.com/codeflow?start=32&length=7)

Do NOT change this. This is required for the Ubuntu 14 build. Cmake 3 is only for the full OSConfig build and that is clearly stated in the main readme.

**Review by MariusNi — CHANGES_REQUESTED** [review]
:clock1:

File: src/adapters/mc/README.md (hunk 1 above)
Comment (MariusNi) [reply src/adapters/mc/README.md]: Also fix the PR description.

**Comment by MariusNi** [comment]
The PR currently states the following, quote:

'Typo fixes and ubuntu 14 installation dependency fix
Upon manual inspection of build process on Ubuntu 14 I found that cmake was at version 2.8.12.2.
Azure-osconfig requires cmake >= 3.2 - and this is fixed by apt-get install cmake3, then cmake will report version3 3.5.1"

This PR is only for the typos. The actual Ubuntu 14 build instructions are correct and must NOT be changed. Full OSConfig requires Cmake 3.2+, OSConfig for MC doesn't. Please revert the line. 
anno

**Comment by MariusNi** [comment]
Also, please fix in the PR description, there were no tests added for this PR so you cannot state this there. Just say that all tests are passing (and validate that).

Speaking of which, did you test building the policy packages on Ubuntu 14 and testing as described in the readme?

---
In reply to: [2429725054](https://github.com/Azure/azure-osconfig/pull/774#issuecomment-2429725054) [](http://example.com/codeflow?ancestors=2429725054)

**Review by MariusNi — APPROVED** [review]
:shipit:

**Review by robertschaedler3 — APPROVED** [review]

**Pull request merged**